src/middleware/jwtValidator.js

```diff
@@ -0,0 +1,33 @@
+let jwt = require('jsonwebtoken');
+const config = require('../database/config/config.js');
+
+let checkToken = (req, res, next) => {
+  let token = req.headers['x-access-token'] || req.headers['authorization']; // Express headers are auto converted to lowercase
+  if (token.startsWith('Bearer ')) {
+    // Remove Bearer from string
+    token = token.slice(7, token.length);
+  }
+
+  if (token) {
+    jwt.verify(token, config.development.password, (err, decoded) => {
+      if (err) {
+        return res.json({
+          success: false,
+          message: 'Token is not valid'
+        });
+      } else {
+        req.decoded = decoded;
+        next();
+      }
+    });
+  } else {
+    return res.json({
+      success: false,
+      message: 'Auth token is not supplied'
+    });
+  }
+};
+
+module.exports = {
+  checkToken: checkToken
```

Comment:
You should try to return a **status code** too with your response. There is a `response` method already defined in `src/helpers/response.js`, use that to handle your responses.